examples/nginx/site-available

```diff
@@ -0,0 +1,52 @@
+# This is an example nginx `sites-available` file to use with maxwell-daemon.
+# Lua scripting is used to communicate with the daemon.
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    location / {
+        # Arbitrate the request's path (defaulting to master).
+        set $maxwell "master";
+        access_by_lua '
```

Comment:
It's just an example, I'd rather not have to include tests and benchmarks for all the sample use cases we come up with, is that a thing people do?